Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Compressors/ASTCCompressor.cpp

```diff
@@ -182,10 +180,12 @@ namespace ImageProcessingAtom
         // Create a context based on the configuration
         astcenc_context* context;
         AZ::u32 blockCount = ((srcImage->GetWidth(0)+ dstFormatInfo->blockWidth-1)/dstFormatInfo->blockWidth) * ((srcImage->GetHeight(0) + dstFormatInfo->blockHeight-1)/dstFormatInfo->blockHeight);
-        AZ::u32 threadCount = AZStd::min(AZStd::thread::hardware_concurrency(), blockCount);
+        AZ::u32 threadCount = AZStd::min(AZStd::thread::hardware_concurrency()/2, blockCount);
```

Comment:
Would this work better for the AssetProcessor if it was always set to 1?

Could we add a Jira to make the ratio configurable in the settings registry.

Reply:
We would want it higher than 1 for AP in case when AP is processing one texture. 
I may add the configurable if makes a difference with performance under different conditions.